salt/netapi/rest_tornado/saltnado.py

```diff
@@ -1100,13 +1100,15 @@ def _disbatch_runner(self, chunk):
         '''
         Disbatch runner client commands
         '''
+        full_return = chunk.pop('full_return', False)
         pub_data = self.saltclients['runner'](chunk)
         tag = pub_data['tag'] + '/ret'
         try:
             event = yield self.application.event_listener.get_event(self, tag=tag)
 
             # only return the return data
-            raise tornado.gen.Return(event['data']['return'])
+            ret = event if full_return else event['data']['return']
```

Comment:
is this what we are doing in other release?

I thought we returned the whole event with data and return in cherrypy?